package.json

```diff
@@ -66,4 +68,4 @@
       "@semantic-release/git"
     ]
   }
-}
+}
```

Comment:
nit: add a newline at the end of the file

Reply:
`Alt + Shift + F` (Format Document) in **VSCode** 😄